Gruntfile.coffee

```diff
@@ -279,6 +302,13 @@ module.exports = (grunt) ->
 
         command: 'rm -rf compiled dist src/{plugins_,}settings.coffee'
 
+      clean_dist:
+        options:
+          stdout: true
+
+        command: 'rm -rf compiled dist src/{plugins_,}settings.coffee'
```

Comment:
Why we need to duplicate this one? 
https://github.com/skroutz/analytics.js/pull/28/files#diff-a35864fcc4749a3b904641b4be9feacfL280